### PR TITLE
chore: Update translations

### PR DESCRIPTION
**Pull request by RuffleBuild**
New Crowdin pull request with translations 🎉

### CODE REVIEW

**Review by kjarosh — CHANGES_REQUESTED** [review]
Huh... a lot of translations have suddenly disappeared :sob:

File: desktop/assets/texts/cs-CZ/preferences_dialog.ftl [hunk 1]
```diff
@@ -23,3 +23,10 @@ theme = Téma
 theme-system = Výchozí systému
 theme-light = Světlá
 theme-dark = Tmavá
+# See for context https://github.com/FeralInteractive/gamemode
+gamemode = Herní režim
```
Comment (kjarosh) [inline desktop/assets/texts/cs-CZ/preferences_dialog.ftl:27]: I'm fairly certain this translation is wrong

**Review by kjarosh — APPROVED** [review]
Everything should be fixed now

File: desktop/packages/linux/rs.ruffle.Ruffle.desktop [hunk 2]
```diff
@@ -17,17 +17,14 @@ GenericName[ar]=مشغل الفلاش
...
 GenericName[es]=Reproductor de Flash
 GenericName[fr]=Lecteur Flash
-GenericName[ja]=Flash Player
+GenericName[ja]=Flashプレイヤー
 GenericName[ko]=플래시 플레이어
 GenericName[nl]=Flash speler
 GenericName[pl]=Odtwarzacz Flash
-GenericName[pt_PT]=Flash Player
-GenericName[ru]=Flash Player
-GenericName[uk]=Flash Player
```
Comment (torokati44) [inline desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: There's no point to removing these, but it doesn't matter either, as they are the same as the source string.

**Comment by torokati44** [comment]
Thank you for fixing things up!

File: desktop/packages/linux/rs.ruffle.Ruffle.desktop (hunk 2 above)
Comment (kjarosh) [reply desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: They will be marked as not translated on Crowdin so that people can translate them properly

Comment (torokati44) [reply desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: I see, okay!

**Pull request merged**